// 2016 misc / generator throw() caught by inner generator
module.exports = () => {
  function * generator() {
    yield * (function * () {
      try {
        yield 'foo';
      }
      catch(e) {
        return;
      }
    }());
    yield 'bar';
  }
  var iter = generator();
  iter.next();
  return iter['throw']().value === 'bar';

};
----------------------------------------------------------
Process spawn error. Is java in the path?
spawn /tmp/faast/882c7796-a2a2-4b16-b485-c841650cc1d5/node_modules/google-closure-compiler-linux/compiler EACCES
